invenio_github/ext.py

```diff
@@ -84,27 +77,28 @@ def connect_signals():
             from .handlers import account_post_init
 
             account_setup_committed.connect(
-                account_post_init,
-                sender=GitHubAPI.remote._get_current_object()
+                account_post_init, sender=GitHubAPI.remote._get_current_object()
```

Comment:
question: do we want these behaviours? 

What happens is the following:

- User connects Github account (invenio-oauthclient)
- A signal is emitted and we start syncing user's repositories, initialising their account.

The alternative is not to sync anything automatically, instead, it's synced on demand (e.g. an HTTP request)